manager/README.md

```diff
@@ -0,0 +1,92 @@
+# Transporter Manager
```

Comment:
for both the manager and agent markdowns I would link back to the transporter readMe